_includes/readmes/express-master/examples.md

```diff
@@ -0,0 +1,30 @@
+# Express examples
+
+This page contains list of examples using Express.
+
+- [auth](./auth) - Authentication with login and password
```

Comment:
Hey @crandmck does the site build process fix these links to point to the express repo or do we need to add something here that does it?

Reply:
Locally it's building it as relative path so I added [replace "pipe"](https://github.com/expressjs/expressjs.com/pull/1178/files#diff-e944bb33b6f0dc3eb77f1ef492e07188R10) in template which is quite ugly.

Will be happy if you can suggest something to avoid this

Reply:
@dougwilson The site build doesn't do it, but @rodion-arr wrote some code in `examples_page/en/starter/examples.md` that does it.  I agree it's not pretty, but offhand I can't think of a better way to do it.